app/src/androidTest/java/org/mozilla/reference/browser/SettingsMenuItemsTest.kt

```diff
@@ -0,0 +1,107 @@
+package org.mozilla.reference.browser
+
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import android.view.View
+import android.view.ViewGroup
+
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.hamcrest.core.IsInstanceOf
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
```

Comment:
Note: the test runner defaults to AndroidJUnit4 so this annotation is redundant.